### DISCUSSION TITLE
Water sounds according to its depth

### DISCUSSION THREAD

**Discussion by misobadev** [Sixze/ALS-Refactored#472] [Q&A]
What do you recommend me to do to add the sounds to the footstep when I go through a lake?

The idea is to have two levels of sounds depending on the depth of the lake, so I can play water sound when I am near the edge of the lake and another sound when the lake is above my knees.

I have studied a little bit the ALS Refactored and I found that every step the character takes calls ALS Footstep Effects

I have created my physical material water and I have also created my surfacetype Water.

I have also added the effect in the FES_Als_Default file.

I have tested with Grass and Concrete and everything worked fine.

The problem is the water, because in my opinion, to meet my requirement, this should have a different treatment.

So here again I reiterate my question, what do you recommend me to do to add the water effect I need using the footstep effect class?

**Comment by Magic-Summer** [comment]
Perhaps a different [physical material](https://github.com/Sixze/ALS-Refactored/discussions/268) for deeper water?
